staff/maider-hernandorena/beers-app/.versions/beers-app-1.0.0/scripts/main.js

```diff
@@ -0,0 +1,50 @@
+// my presentation logic
```

Comment:
you should use 'use strict' at the beginning of the js page.

Reply:
Changed in all files! :D